2022-12-19 01:33:26.959 +02:00 [ERR] Error: {"ClassName":"System.Exception","Message":"TaskResult","Data":null,"InnerException":null,"HelpURL":null,"StackTraceString":"   at SpendLess.UnitTests.Tests.<>c.<<TaskResultMethodsWithExceptionsAreLoggedByInterceptor>b__38_0>d.MoveNext() in C:\\Users\\viliu\\Documents\\GitHub\\SpendLess\\SpendLess\\UnitTests\\Tests.cs:line 327\r\n--- End of stack trace from previous location ---\r\n   at SpendLess.Server.Interceptor.UnhandledExceptionLogger.InterceptAsync[T](Task`1 task) in C:\\Users\\viliu\\Documents\\GitHub\\SpendLess\\SpendLess\\Server\\Interceptor\\UnhandledExceptionLogger.cs:line 63","RemoteStackTraceString":null,"RemoteStackIndex":0,"ExceptionMethod":null,"HResult":-2146233088,"Source":"Tests","WatsonBuckets":null}
